docs/primer/releases_and_records.md

```diff
@@ -23,20 +23,20 @@ Open Contracting Data Standard (OCDS) data is published and updated over time us
 
 The only thing that can be truly called “OCDS data” is JSON data that validates against the OCDS schema. There are two schemas for OCDS data: “releases” and “records.”
 
-OCDS publishers are encouraged to use releases and records to publish data in near real-time and to provide a change history. Tracking change over time is important to users of contracting data because there can be many changes in the life of a contracting process. Knowing how and when certain attributes have changed can help users to understand the process, identify inefficiencies, and spot red flags for corruption.
+OCDS publishers are encouraged to use releases and records to publish data in near real-time and to provide a change history. Tracking change over time is important to users of contracting data because there can be many changes in the life of a contracting (or planning) process. Knowing how and when certain attributes have changed can help users to understand the process, identify inefficiencies, and spot red flags for corruption.
```

Comment:
We don't define planning process in the primer. Should we just leave these as-is?

Reply:
I've updated https://standard.open-contracting.org/staging/1.2-dev/en/schema/reference/#planning (mainly reused the description of `planning`), removed the "or planning" in the primer and added "or planning" in `releases` in `record-package-schema.json` in https://github.com/open-contracting/standard/pull/1513/commits/bdfd38f4d23ec645ed1d9784d8d143cc3f7e54a8.

I'm not sure `ocid` needs updating - the planning process is in the second sentence there?

Reply:
> I'm not sure ocid needs updating - the planning process is in the second sentence there?

Oh, yes, I was lazy and stopped reading after the first sentence :)